src/Sentry.Hangfire/GlobalConfigurationExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using Hangfire;
+using Sentry.Extensibility;
+
+namespace Sentry.Hangfire;
+
+/// <summary>
+/// Hangfire Extensions for <see cref="GlobalConfigurationExtensions"/>.
+/// </summary>
+public static class GlobalConfigurationExtensions
+{
+    /// <summary>
+    /// Uses Sentry
+    /// </summary>
+    /// <param name="configuration"></param>
+    /// <returns></returns>
+    public static IGlobalConfiguration UseSentry(this IGlobalConfiguration configuration)
+    {
+        configuration.UseFilter(new SentryServerFilter());
+        return configuration;
```

Comment:
It might be better to chain these constructors by using `: this(null, null)` here. Otherwise every time you make changes to one, you have to remember to make equivallent changes to the other, which is quite dangerous when one of them is used for testing and the other for production. 

It means you'd need the `hub` parameter on the overload nullable and maybe do something like:
```csharp
hub ??= HubAdapter.Instance;
```

Similarly, IDiagnosticLogger would have to be nullable but the `SentryServerFilter` is able to deal with that anyway.